File: test/rpcc.js
var zerorpc = require("zerorpc");

var client = new zerorpc.Client();
client.connect("tcp://127.0.0.1:4242");

client.invoke("echo", [1,2,3], function(error, res, more) {
    console.log(res);
});
client.invoke("echo", {1:1,2:2,a:3}, function(error, res, more) {
    console.log(res);
});

File: test/rpcc.lua
local client = require "zerorpc.client"
local S = client.new("tcp://127.0.0.1:4242")

r = S:call("echo",{1,2,3})
if type(r) == 'table' then
    table.foreach(r,print)
else
    print(r)
end
--r = S:call("echo",{1,2,a=3})
--print(r)
--table.foreach(r,print)
S:close()